libs/template/presets/fulfillment/experience/pages/login-page.ts

```diff
@@ -0,0 +1,26 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
+import { Size } from '@spryker-oryx/utilities';
+
+export const fulfillmentLoginPage: ExperienceComponent = {
+  id: 'fulfillment-login-page',
+  type: 'Page',
+  meta: {
+    title: 'Login Page',
+    route: '/login',
+    description: 'Login Page Description',
+  },
+  options: {
+    rules: [
+      { layout: 'flex', justify: 'center' },
+      {
+        query: { breakpoint: Size.Sm },
+        width: '100%',
+      },
```

Comment:
```suggestion
      { layout: 'list'},
```

Reply:
Here's btw the inspiration to drop the picking login component:

```ts
options: {
    rules: [
      {
        layout: 'list',
        height: '100vh',
        gap: '40px',
        style: 'align-content: center',
      },
    ],
  },
  components: [
    {
      type: 'oryx-content-image',
      id: 'site-logo',
      content: {
        data: {
          graphic: 'logo',
          label: 'Fulfillment App based on Oryx',
        },
      },
      options: {
        rules: [{ height: '82px' }],
      },
    },
    {
      type: 'oryx-content-text',
      content: {
        data: {
          text: `<h3>${i18n('welcome-please-log-in-to-start-picking')}</h3>`,
        },
      },
    },
    {
      type: 'oryx-auth-login',
      options: {
        enableRememberMe: false,
      },
    },
  ],
```

Reply:
Looks great, but we decided to finish it in separate ticket.